src/infill.cpp

```diff
@@ -4,9 +4,20 @@
 #include "utils/polygonUtils.h"
 #include "utils/logoutput.h"
 
-namespace cura {
+/*!
+ * Function which returns the scanline_idx for a given x coordinate
+ *
+ * For negative \p x this is different from simple division.
+ *
+ * \warning \p line_width is assumed to be positive
+ *
+ * \param x the point to get the scansegment index for
+ * \param line_width the width of the scan segments
+ */
+static inline int computeScanSegmentIdx(int x, int line_width);
```

Comment:
I guess this is just a matter of taste...

If you put it in the header file with private scope then other modules won't be able to acces the function anyway.